code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -40,7 +43,7 @@
 				filling.icon_state = "[icon_state]100"
 
 		filling.icon += mix_color_from_reagents(reagents.reagent_list)
-		overlays += filling
+		underlays += filling
```

Comment:
To address the complaints that coloured bottles (mainly the reagent one) didn't tint the reagents inside it, also because most of the bottle sprites have these little 'label' pixels that would normally just get hidden